package.json

```diff
@@ -84,16 +84,16 @@
     "@semantic-release/github": "^8.0.4",
     "@semantic-release/npm": "^9.0.1",
     "@semantic-release/release-notes-generator": "^10.0.3",
-    "@storybook/addon-controls": "^6.4.19",
-    "@storybook/addon-docs": "^6.4.19",
-    "@storybook/addon-knobs": "^6.4.0",
-    "@storybook/addon-viewport": "^6.5.8",
-    "@storybook/addons": "^6.4.19",
+    "@storybook/addon-controls": "~6.4.20",
```

Comment:
storybook 6.5.x introduces issues with persisting the storybook controls in the url, which is something we rely on for screenshot testing